.github/workflows/build.yml

```diff
@@ -55,9 +55,9 @@ jobs:
     outputs:
       versions: ${{ steps.generate-matrix.outputs.versions }}
     steps:
-    - name: Select 2 most recent LTS versions of Node.js
+    - name: Select 3 most recent LTS versions of Node.js
```

Comment:
maybe instead of just hardcoding this number we could fetch version numbers that are not marked with `its: false` and whose `eol` timestamp is below the current date?

Reply:
I'm not fully sure how to add the current date in the jq command, but there's also a dedicated GH action that can do all of this: https://github.com/marketplace/actions/node-lts-versions
Maybe it's easier to just add that?

Reply:
yes, lets try the action, maybe It already does all we need

Reply:
Okay, seems like it works as expected. It has options for `active` which is what we want, and also `lts` which excludes versions that are current but not yet LTS (in this case Node 22)